src/App.js

```diff
@@ -51,7 +51,8 @@ class AppFrame extends Component {
 
   onSignIn(status, data) {
     this.setState({loggedIn: status, userData: data});
-    if (data.basic.username == "adminacc") {this.setState({admin: true})};
+    // Needs to be set back to False when someone signs in that is not an admin if an admin has previously signed in
+    data.admin ? this.setState({admin: true}) : this.setState({admin: false});
```

Comment:
Since state updates are asynchronous, we may want to determine this beforehand and set the state only once in this function.